types/underscore/underscore-tests.ts

```diff
@@ -697,11 +698,11 @@ interface StringRecordAugmentedList extends _.List<StringRecord> {
 
 const stringRecordAugmentedList: StringRecordAugmentedList = { 0: { a: 'a', b: 'c' }, 1: { a: 'b', b: 'b' }, 2: { a: 'c', b: 'a' }, length: 3, notAListProperty: true };
 const stringRecordList: _.List<StringRecord> = stringRecordAugmentedList;
-declare const twoDimensionStringRecordList: _.List<_.List<StringRecord>>;
-declare const threeDimensionStringRecordList: _.List<_.List<_.List<StringRecord>>>;
-declare const fourDimensionStringRecordList: _.List<_.List<_.List<_.List<StringRecord>>>>;
-declare const zeroAndOneDimensionStringRecordArray: (StringRecord | StringRecord[])[];
-declare const zeroOneTwoDimensionStringRecordArray: (StringRecord | StringRecord[] | StringRecord[][])[];
+declare const level2RecordList: _.List<_.List<StringRecord>>;
+declare const level3RecordList: _.List<_.List<_.List<StringRecord>>>;
+declare const level4RecordList: _.List<_.List<_.List<_.List<StringRecord>>>>;
+declare const twoLevelRecordArray: (StringRecord | StringRecord[])[];
+declare const threeLevelRecordArray: (StringRecord | StringRecord[] | StringRecord[][])[];
```

Comment:
Yeah I see what you mean. Maybe `maxLevel2RecordArray` and `maxLevel3RecordArray` would be more consistent?

Reply:
Works for me, thanks!